lib/inferno/repositories/session_data.rb

```diff
@@ -27,7 +27,7 @@ def load(test_session_id:, name:, type: 'text')
 
         case type.to_s
         when 'oauth_credentials'
-          DSL::OAuthCredentials.new(JSON.parse(raw_value))
+          DSL::OAuthCredentials.new(JSON.parse(raw_value || '{}'))
```

Comment:
Any concerns with this giving the idea that a OAuth Credential was provided even when one actually wasn't?

Any test accessing this 'oauth_credential' after will get a token with 'token_retrieval_time` filled:

https://github.com/inferno-framework/inferno-core/blob/fb1b969c9eb7ca628f5255abf4859e60f84334c9/lib/inferno/dsl/oauth_credentials.rb#L34

Reply:
That's a good catch.